common/djangoapps/student/tests/test_models.py

```diff
@@ -103,3 +103,16 @@ def test_users_enrolled_in_all(self):
             CourseEnrollment.objects.users_enrolled_in(self.course.id, include_inactive=True)
         )
         self.assertListEqual([self.user, self.user_2], all_enrolled_users)
+
+    def test_users_enrolled_in_without_unauth_or_lti(self):
+        """
+        CourseEnrollment.users_enrolled_in should not return unauthenticated or lti users.
+        """
+        CourseEnrollmentFactory.create(user=self.user, course_id=self.course.id, is_active=True)
+        self.user_n = UserFactory.create(email='test@example.com')
```

Comment:
This doesn't need to be an instance method, does it?
If so, let's just make this a local variable, instead of assigning it to `self`.